arches/install/arches-templates/project_name/webpack/webpack-utils/build-template-filepath-lookup.js

```diff
@@ -21,7 +21,7 @@ const _buildTemplateFilePathLookup = function(path, outerAcc, templateDirectoryP
 
             return { 
                 ...acc, 
-                [Path.join('templates', filename)]: Path.resolve(__dirname, Path.join(outerPath, subPath))
+                [Path.join('templates', filename).replace(/\\/g, '/')]: Path.resolve(__dirname, Path.join(outerPath, subPath))
```

Comment:
wouldn't this change break non-windows installations?

Reply:
_Copying from other comment because it's the same_

I'm not a Linux expert but my logic here is that Linux paths do not have backslashes and, since this only replaces existing backslashes with forward slashes, it doesn't affect any paths on anything but Windows environments.

If there are case where this assumption about Linux paths doesn't work then this could be changed to include a environment condition to guaranteed only affecting Windows environments.
Ex.
if(process.platform === 'win32'){

I did spin up a Ubuntu VM and was able to run the application without issue.

Reply:
yep my mistake. When I first looked at this I thought you were replacing slashes the other way.